### PR TITLE
Allow configurable css and pictures directories

### PR DESCRIPTION
**Pull request by xavierba**
With these patches, css and pictures directories locations are made configurable.
The goal is to allow to relocate them as described in issue #172.

This has been tested lightly for now, hopefully I haven't missed anything.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -61,6 +61,8 @@ if test "$fhs" = "yes"; then
     execcgidir=$libdir/sympa/cgi
     expldir=$localstatedir/lib/sympa/list_data
     staticdir=$localstatedir/lib/sympa/static_content
+    cssdir=$dlocalestatedir/lib/sympa/static_content/css
+    picturesdir=$dlocalestatedir/lib/sympa/static_content/pictures
```
Comment (ikedas) [inline configure.ac]: s/dlocalestatedir/localestatedir/

Comment (ikedas) [reply configure.ac]: Ouch, comment has typo.

s/dlocalestatedir/localstatedir/

Comment (xavierba) [reply configure.ac]: Oops, sorry for the silo typo :-(
I fixed the commit and force-pushed.

File: configure.ac [hunk 2]
```diff
@@ -61,6 +61,8 @@ if test "$fhs" = "yes"; then
     execcgidir=$libdir/sympa/cgi
     expldir=$localstatedir/lib/sympa/list_data
     staticdir=$localstatedir/lib/sympa/static_content
+    cssdir=$localestatedir/lib/sympa/static_content/css
+    picturesdir=$localestatedir/lib/sympa/static_content/pictures
```
Comment (ikedas) [inline configure.ac]: s/localestatedir/localstatedir/

Comment (xavierba) [reply configure.ac]: It seems I can't get these 2 lines right... :-( I fixed and force-pushed.

**Pull request merged**